python/cugraph/cugraph/tests/link_prediction/test_overlap.py

```diff
@@ -144,93 +152,95 @@ def extract_two_hop(read_csv):
+        cu_M[EDGE_ATT_COL] = cudf.Series(M[EDGE_ATT_COL])
+
     G1 = cugraph.Graph()
+    weight = EDGE_ATT_COL if use_weight else None
     G1.from_cudf_edgelist(
-        cu_M, source=["src_0", "src_1"], destination=["dst_0", "dst_1"]
+        cu_M,
+        source=[MULTI_COL_SRC_0_COL, MULTI_COL_SRC_1_COL],
+        destination=[MULTI_COL_DST_0_COL, MULTI_COL_DST_1_COL],
+        weight=weight,
     )
 
-    vertex_pair = cu_M[["src_0", "src_1", "dst_0", "dst_1"]]
+    vertex_pair = cu_M[
+        [
+            MULTI_COL_SRC_0_COL,
+            MULTI_COL_SRC_1_COL,
+            MULTI_COL_DST_0_COL,
+            MULTI_COL_DST_1_COL,
+        ]
+    ]
     vertex_pair = vertex_pair[:5]
 
-    df_res = cugraph.overlap(G1, vertex_pair)
-    df_plc_exp = exp_overlap(G1, vertex_pair)
-
-    df_plc_exp = df_plc_exp.rename(
-        columns={
-            "0_src": "0_source",
-            "0_dst": "0_destination",
-            "1_src": "1_source",
-            "1_dst": "1_destination",
-        }
-    )
-    overlap_res = df_res["overlap_coeff"].sort_values().reset_index(drop=True)
-    overlap_plc_exp = df_plc_exp["overlap_coeff"].sort_values().reset_index(drop=True)
-    assert_series_equal(overlap_res, overlap_plc_exp)
-
+    df_multi_col_res = cugraph.overlap(G1, vertex_pair, use_weight=use_weight)
     G2 = cugraph.Graph()
-    G2.from_cudf_edgelist(cu_M, source="src_0", destination="dst_0")
-    df_exp = cugraph.overlap(G2, vertex_pair[["src_0", "dst_0"]])
+    G2.from_cudf_edgelist(
+        cu_M, source=MULTI_COL_SRC_0_COL, destination=MULTI_COL_DST_0_COL, weight=weight
+    )
+    df_single_col_res = cugraph.overlap(
+        G2, vertex_pair[[MULTI_COL_SRC_0_COL, MULTI_COL_DST_0_COL]]
+    )
 
     # Calculating mismatch
-    actual = df_res.sort_values("0_first").reset_index()
-    expected = df_exp.sort_values("first").reset_index()
-    assert_series_equal(actual["overlap_coeff"], expected["overlap_coeff"])
+    actual = df_multi_col_res.sort_values("0_src").reset_index()
+    expected = df_single_col_res.sort_values(VERTEX_PAIR_FIRST_COL).reset_index()
+    assert_series_equal(actual[OVERLAP_COEFF_COL], expected[OVERLAP_COEFF_COL])
 
 
 @pytest.mark.sg
-def test_weighted_exp_overlap():
+def test_weighted_overlap():
     karate = UNDIRECTED_DATASETS[0]
-    G = karate.get_graph()
-    with pytest.raises(ValueError):
-        exp_overlap(G)
-
     G = karate.get_graph(ignore_weights=True)
-    use_weight = True
     with pytest.raises(ValueError):
-        exp_overlap(G, use_weight=use_weight)
-
+        cugraph.overlap(G, use_weight=True)
 
-@pytest.mark.sg
-def test_invalid_datasets_overlap():
-    karate = UNDIRECTED_DATASETS[0]
-    df = karate.get_edgelist()
-    df = df.add(1)
-    G = cugraph.Graph(directed=False)
-    G.from_cudf_edgelist(df, source="src", destination="dst")
+    G = karate.get_graph(ignore_weights=True)
     with pytest.raises(ValueError):
-        cugraph.overlap(G)
+        cugraph.overlap(G, use_weight=True)
```

Comment:
same question here and elsewhere (sorensen, etc.): it looks like this is just checking for ValueError for the same unweighted graph twice.